.github/scripts/enable-ccache.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -e # exit on error
+set -x # echo on
+set -o pipefail # fail of any command in pipeline is an error
+
+if [ -z "$GITHUB_WORKSPACE" ]; then
+  DIR=`pwd`
```

Comment:
perhaps the root folder should be detected, similar as it is done mingw-build.

Reply:
Can you please point me to a specific line of code so I'll understand what you are referring to?

Reply:
If you mean `dirname ${BASH_SOURCE[0]}` then this code would look like:

```bash
DIR=`dirname ${BASH_SOURCE[0]}`../../../
if [[ ! -z "$GITHUB_WORKSPACE" ]]; then
  DIR=`cygpath "$DIR"`
fi
```

Reply:
Changed.